app/Enrollment.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Enrollment extends Model {
```

Comment:
How much would it suck to change this to something other than `enrollments`? Just wondering about confusion down the line if we do more enrollment-centric features (section count / enrollment size / etc)

Reply:
It seems doable. Do you want to do it now, or just wondering for a future change? Have something in mind? 

Canvas calls it "enrollment," regardless of role: student, instructor, ta. Blackboard calls it "course membership," but we already have members for groups so that might be confusing. Is there a standard UMN or PeopleSoft term that's used?

Reply:
As far as I know, they're all called instructors in peoplesoft, and then they have different roles. I can't think of another term in use at the U, and glancing through the LDW I don't see anything. It's probably not worth the hassle to refactor, since as far as I can see we're not exposing that terminology to the end user anywhere?